app/src/include/firebase/app.h

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-#ifndef FIREBASE_APP_CLIENT_CPP_SRC_INCLUDE_FIREBASE_APP_H_
-#define FIREBASE_APP_CLIENT_CPP_SRC_INCLUDE_FIREBASE_APP_H_
+#ifndef FIREBASE_APP_SRC_INCLUDE_FIREBASE_APP_H_
+#define FIREBASE_APP_SRC_INCLUDE_FIREBASE_APP_H_
 
 #include "firebase/internal/platform.h"
 
```

Comment:
📝 Documentation warning: \
`warning: The following parameter of firebase::AppOptions::LoadFromJsonConfig(const char *config, AppOptions *options=nullptr) is not documented:`
`%s`  parameter 'config'<hidden value="hidden-doxygen-comment-tag"></hidden>